### DISCUSSION TITLE
Firopow plans?

### DISCUSSION THREAD

**Discussion by dparadox** [coinfoundry/miningcore#1204] [Q&A]
Hi Everyone, 

Is there any plans for adding Firopow support?

**Comment by seperatis** [comment]
@dparadox 
This is NOT bug, it's a Feature Request and belongs in the Discussions @oliverw

@dparadox and everyone else, please refrain from using the Issue reporting to ask questions! This is what Discussions are for. ISSUES are for reporting bugs or other problems with the software itself. Not for questions or feature requests, etc.

Thank you.

**Comment by dparadox** [comment]
Oops, that was the option I was looking for, sorry I didn't see it before.
Closing this.

**Comment by oliverw** [comment]
Firo is on the (loong) list 😄
